yb-voyager/cmd/exportSchema_test.go

```diff
@@ -102,3 +107,199 @@ func TestShardingRecommendations(t *testing.T) {
 		strings.ToLower(sqlInfo_table3.stmt))
 	assert.Equal(t, matchTable, false)
 }
+
+// Test export schema after running assessment internally - case when assess-migration is run before export-schema
+// Expectation: export-schema should export with no internal assess-migration cmd invokation
```

Comment:
> export-schema should export with no internal assess-migration cmd invokation

how are we asserting this? perhaps you should check the msr flag like in the other tests?

Reply:
done